Modules/_sqlite/connection.c

```diff
@@ -1442,6 +1457,173 @@ pysqlite_connection_iterdump(pysqlite_Connection* self, PyObject* args)
     return retval;
 }
 
+#ifdef HAVE_BACKUP_API
+static PyObject *
+pysqlite_connection_backup(pysqlite_Connection* self, PyObject* args, PyObject* kwds)
+{
+    PyObject* target = NULL;
+    const char* filename;
+    int pages = -1;
+    PyObject* progress = Py_None;
+    char* name = "main";
+    PyObject* retval = NULL;
+    int rc;
+    int cberr = 0;
+    int sleep = 250;
+    sqlite3 *bckconn;
+    sqlite3_backup *bckhandle;
+    static char *keywords[] = {"target", "pages", "progress", "name", "sleep", NULL};
+
+    if (!PyArg_ParseTupleAndKeywords(args, kwds, "O|$iOsi:backup", keywords,
```

Comment:
See other comment, Format ``d`` accepts ints and floats and converts them to double. Multiple the double with 1000 and cast it to int ``sqlite3_sleep(sleep)``

Reply:
Ok, will do!